evalml/preprocessing/data_splitters/no_split.py

```diff
@@ -29,6 +29,15 @@ def get_n_splits():
         """
         return 0
 
+    @property
```

Comment:
Omega nit-pick but can we make this abstract on `BaseCrossValidator` to so that we don't forget to implement it for any future classes we might add?

Reply:
@angela97lin `BaseCrossValidator` is a class we take from sklearn. Not sure if we can add an abstract to this unless we create our own `BaseCrossValidator` class with this attribute?